### ISSUE TITLE
Passing screen size ?

### ISSUE THREAD

**Issue by florian-guillemard** [toufikzitouni/flutter-bidirectional_scrollview_plugin#3]
Hi,

I was looking for something you did! Thanks!

I tested it without a problem but now, in my case, i'm trying to display something which needs to be related to the screen size and I can't figuring out how I can make it..

Can you make a simple example for passing argument in the build() ?

In the initState, we can't have the screen size because it's not already build :/

Thanks for your help and for your good work!

Regards

**Comment by sarangkartikey50** [comment]
Try using it inside a container and provide desired height. It worked for me!

**Comment by florian-guillemard** [comment]
Yes, it is what I did.. I had to adapt the lib to be hot-loaded by a child too and init the offset !
Thank you for your help @sarangkartikey50 ;)

**Comment by madisonkwynn** [comment]
I also have a screen size issue. I am a developer for a fully animated foster children app and I have a barrier of not having the drag and positioned widgets available when the user scrolls right. 

As I am using a positioned widget, it can only position itself in the screen area available when the phone loads. But however, I want the positioned widget to be at the far right, which takes some scrolling to get there. This is a major problem for my app because most of it is horizontal scrolling.

It also appears that you cannot drag objects to the right of the screen as well.

Is list view the only possible way to have items at the end of the bidirectional scrolling (right side). I can make the middle list view items invisible and positionally stack it that way? 

I have also tried a stateful widget with offset and initState and as soon as I offset anything outside the phone screen size, the widget or drag widget disappears.

**Comment by madisonkwynn** [comment]
Code fixed. Please disregard the comment up there.

In the code, you can use bidirectional or ListView:

Widget build(BuildContext context) {
    return Scaffold (body: SafeArea(
      child: ListView(
        scrollDirection: Axis.horizontal,
        children: <Widget>[
        Stack(
            children: <Widget>[
              Container(...
              Container(...) end of stack
       Positioned
              left: 2150.0,
              bottom: 140.0,
              child: Container(...
               ...) end of ListView
          
To allow positional widgets at the end of Horizontal Scrolling, add it to the end of the list.
